src/components/card/_macro-options.md

```diff
@@ -18,3 +12,21 @@
 | largeSrc       | string | false    | Path to the retina version of the image                                                                              |
 | alt            | string | false    | The HTML `alt` tag to explain the appearance and function of the image. Not required if the image is only decorative |
 | placeholderURL | string | false    | Optional base path for placeholder image                                                                             |
+
+## Title
+
+| Name         | Type   | Required | Description                                                                   |
+| ------------ | ------ | -------- | ----------------------------------------------------------------------------- |
+| text         | string | true     | The text for the title of the card heading                                    |
+| headingLevel | int    | false    | Number used to determine the heading level of the card title. Defaults to `2` |
+| classes      | string | false    | Font size classes for the card heading. Defaults to `ons-u-fs-m`              |
+| url          | string | true     | The URL for the title link `href` attribute                                   |
+| id           | string | true     | The HTML `id` attribute for the card heading                                  |
```

Comment:
Again "title" would make more sense with the other desciptions

Reply:
updated